assopy/admin.py

```diff
@@ -99,7 +99,6 @@ def get_formset(self, request, obj=None, **kwargs):
 class OrderAdminForm(forms.ModelForm):
     method = forms.ChoiceField(choices=(
         ('admin', 'Admin'),
-        ('paypal', 'PayPal'),
```

Comment:
This could be based on assopy/models.py::ORDER_PAYMENT

+ deleting probably won't cause any issues, but I'd rather go with `('paypal', 'Paypal DEPRECATED')` instead here.
Since there's still >800 orders with paypal in db, some of them from 2018, opening and saving a form with old order might cause some inconsistencies.

Reply:
You're 100% right - I've updated this to use `Order.ORDER_PAYMENT` values and `admin` and renamed `Paypal` to `Paypal (deprecated)`